__tests__/pet.tests.js

```diff
@@ -0,0 +1,170 @@
+const Pet = require("../src/pet");
+
+describe("constructor", () => {
+    it('returns a new object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+    });
+    it('sets intial age to 0, hunger of 0, fitness of 10', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toEqual(0);
+        expect(pet.hunger).toEqual(0);
+        expect(pet.fitness).toEqual(10);
+    });
+});
+
+describe('growUp', () => {
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+        pet.age = 30;
+        expect(() => pet.growUp()).toThrow('Your pet is no longer alive :(');
+    });
+    
+    
+    it('sets growUp method to increase age prop by 1, hunger by 5, fitness down by 3', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+
+        expect(pet.age).toEqual(1);
+        expect(pet.hunger).toEqual(5);
+        expect(pet.fitness).toEqual(7);
+
+        pet.growUp();
+
+        expect(pet.age).toEqual(2);
+        expect(pet.hunger).toEqual(10);
+        expect(pet.fitness).toEqual(4);
+    }); 
+});
+
+describe('walk', () => {
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+        pet.age = 30;
+        expect(() => pet.walk()).toThrow('Your pet is no longer alive :(');
+    });
+   
+    it('walk increases fitness by 4', () => {
+        const pet = new Pet('Fido');
+        pet.fitness = 1;
+
+        pet.walk();
+
+        expect(pet.fitness).toEqual(5);
+    });
+    it('walk increases fitness to a max of 10', () => {
+        const pet = new Pet('Fido');
+        pet.walk(); //initial fitness 10, after walk, still 10;
+
+        expect(pet.fitness).toEqual(10);
+    });
+});
+
+describe('feed', () => {
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+        pet.age = 30;
+        expect(() => pet.feed()).toThrow('Your pet is no longer alive :(');
+    });
+
+    it('feed decreases hunger by 3', () => {
+        const pet = new Pet('Fido');
+        pet.hunger = 5;
+
+        pet.feed();
+
+        expect(pet.hunger).toEqual(2);
+    });
+    it('feed decreases hunger by 3 but not below 0', () => {
+        const pet = new Pet('Fido');
+        pet.hunger = 2;
+
+        pet.feed();
+
+        expect(pet.hunger).toEqual(0); //without limit it would be -1;
+    });
+});
+
+describe('checkUp', () => {
+    it('lets you know how the pet is feeling - best case', () => {
+        const pet = new Pet('Fido');
+        pet.checkUp();
+        expect(pet.checkUp()).toBe('I feel great'); // default values;
```

Comment:
As above, should use a blank line before your assertions start, and the comment isnt needed